### DISCUSSION TITLE
Name template clarification

### DISCUSSION THREAD

**Discussion by rrrnld** [Akylas/OSS-DocumentScanner#236] [Q&A]
Hi,

thanks for the app. I've been searching for a floss document scanning app for quite a while and the app works really nicely.

I am slightly confused about the naming options. How does the template get filled? There is no good explanation of what the different parts mean. I would have expected something like https://strftime.org/ or what the `date` unix util accepts (with a link to its docs), but it seems to be something else. On top of that for some reason there are two different naming schemes.

I've tried adding the "iso" keyword, but `s` seems to be replaced by the single-digit current second, while the "i" and "o" seem to be left as-is. What I'm trying to do is create new document that by default is prefixed with `date +%Y-%m-%d`, nothing more. How would I do that? Thanks for any help, and again thanks for the app.

**Comment by farfromrefug** [comment]
@heyarne it is using this format https://day.js.org/docs/en/display/format.
Now in the dialog where you set the template there is a "here" text" which should link to that doc. But i got reports that the link does not work for some people.
Could you share a screenshot of that dialog?
Now `iso` should work. It is a bug if it does not. Will try it now.
As for what you want:  `[date ]+YY-MM-DD` should do it

**Reply by rrrnld** [reply]
I'm running 1.9.4.80. Where should I expect the link?

![signal-2024-06-17-173033_003](https://github.com/Akylas/OSS-DocumentScanner/assets/525780/9fecb6e5-f6dd-484a-a8cc-b47ea965e27a)
![signal-2024-06-17-173033_002](https://github.com/Akylas/OSS-DocumentScanner/assets/525780/21d220e5-c6f2-4e4e-b478-4cc103ce40a0)

**EDIT** Nevermind, I think I found it: "You can find more here" should link the last word. It doesn't work for me unfortunately.

Do I have to copy the string you suggested? If yes, where does the `[date ]`  prefix come from, and what does it do? The other date format strings I know work like this: All date parts are prefixed using `%`, for example `%Y` is the year. Everything that's not prefixed with `%` is not replaced by anything. This here works differently and matches on one or more letters, yes? Sorry for all of those questions, but as you can see I'm very confused and don't know where else to find answers.

**Comment by farfromrefug** [comment]
@heyarne yes it is  that dialog. 
About your template sorry i made a mistake, i dont know why but i read `date` as you wanting to actually have the word `date` :s 
So first about the template format read https://day.js.org/docs/en/display/format you ll find everything you need. `[]` is used to escape and ensure you write the actual characters and not transform them.
Will look at the issue with the here text not working. Need to be able to reproduce it
